### PR TITLE
Add Python 3.13 to the testing

### PR DESCRIPTION
**Pull request by cclauss**
* https://www.python.org/downloads/release/python-3130/
* https://pythoninsider.blogspot.com/2024/10/python-3130-final-released.html
* https://devguide.python.org/versions/
* https://docs.djangoproject.com/en/stable/faq/install/#what-python-version-can-i-use-with-django

### CODE REVIEW

**Review by Nusnus — APPROVED** [review]
Thank you!

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -13,18 +13,20 @@ jobs:
       fail-fast: false
       matrix: # https://docs.djangoproject.com/en/stable/faq/install/#what-python-version-can-i-use-with-django
         django-version: ["3.2", "4.2", "5.0", "5.1"]
-        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12', 'pypy-3.10']
+        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13', 'pypy-3.10']
```
Comment (auvipy) [inline .github/workflows/test.yml:16]: I think support for python 3.8 is prematurely dropped as we are going to support python 3.8 with celery 5.5

Comment (auvipy) [inline .github/workflows/test.yml:16]: also inclusion of python 3.13 was not done across the board

Comment (cclauss) [reply .github/workflows/test.yml:16]: As the title says, this pr adds it to the tests, but does not declare full compatibility.

**Pull request merged**